usecase/github_user.go

```diff
@@ -0,0 +1,25 @@
+//Package usecase processes the information from repository, making any needed calculations, data transformation, etc.
+//If it recieved data it's from controllers, coming from delivery layer
+package usecase
```

Comment:
Cool usage of package scope comments.

A little typo:

```suggestion
//Package usecase processes the information from repository, making any needed calculations, data transformation, etc.
//If it received data it's from controllers, coming from delivery layer
package usecase
```